.travis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
     - name: 'Integration Tests Linux'
       script:
-        - make test-integration
+        - TEST_PRUNE_WITH_IMAGE=true make test-integration
```

Comment:
maybe better run it separately after the main test suits finished to avoid re-downloading images?

Reply:
What do you mean? The test of `srcd prune --with-images` command is already done as last https://github.com/src-d/engine/pull/405/files#diff-b67911656ef5d18c4ae36cb6741b7965R36.

Reply:
ahhhhhh. Sorry. Thanks!